spec/dummy/vendor/assets/javascripts/es5-shim.js

```diff
@@ -0,0 +1,1596 @@
+/*!

```

Comment:
@Johnnycus We add dependencies via NPM. Please remove this file and add via npm. See https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/package.json